dashboard/app/helpers/levels_helper.rb

```diff
@@ -10,8 +10,6 @@ def build_script_level_path(script_level)
     case script_level.script_id
     when Script::HOC_ID
       script_puzzle_path(script_level.script, script_level.chapter)
-    when Script::TWENTY_HOUR_ID
-      script_level_path(script_level.script, script_level)
     when Script::EDIT_CODE_ID
```

Comment:
Yep, eventually this whole `case` statement will go away.  The groundwork is in https://github.com/code-dot-org/code-dot-org/pull/1389 which removes the remaining legacy default/*.yml script definitions.